### PR TITLE
Minor version 4.28

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- Chromograph integration for displaying PNGs in case-page
- Add VAF to cancer case general report, and remove some of its unused fields
- Variants filter compatible with genome browser location strings
- Support for custom public igv tracks stored on the cloud
- Add tests to increase testing coverage
- Update case variants count after deleting variants
- Update IGV.js to latest (v2.7.4)
- Bypass igv.js CORS check using `https://github.com/Rob--W/cors-anywhere`
- Documentation on default and custom IGV.js tracks (admin docs)
- Lock phenomodels so they're editable by admins only
- Small case group assessment sharing
- Tutorial and files for deploying app on containers (Kubernetes pods)
- Canonical transcript and protein change of canonical transcript in exported variants excel sheet
- Support for Font Awesome version 6
- Submit to Beacon from case page sidebar
- Hide dismissed variants in variants pages and variants export function
- Systemd service files and instruction to deploy Scout using podman
### Fixed
- Bugfix: unused `chromgraph_prefix |tojson` removed
- Freeze coloredlogs temporarily
- Marrvel link
- Don't show TP53 link for silent or synonymous changes
- OMIM gene field accepts any custom number as OMIM gene
- Fix Pytest single quote vs double quote string
- Bug in gene variants search by similar cases and no similar case is found
- Delete unused file `userpanel.py`
- Primary transcripts in variant overview and general report
- Google OAuth2 login setup in README file
- Redirect to 'missing file'-icon if configured chromogram file is missing
- Javascript error in case page
- Fix compound matching during variant loading for hg38
- Cancer variants view containing variants dismissed with cancer-specific reasons
- Zoom to SV variant length was missing IGV contig select
- Tooltips on case page when case has no default gene panels
### Changed
- Save case variants count in case document and not in sessions
- Style of gene panels multiselect on case page
- Collapse/expand main HPO checkboxes in phenomodel preview
- Replaced GQ (Genotype quality) with VAF (Variant allele frequency) in cancer variants GT table
- Allow loading of cancer cases with no tumor_purity field
- Truncate cDNA and protein changes in case report if longer than 20 characters

**Review:**
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Checklist of features that should verified before merging and deploying:

### Added
- [ ] Chromograph integration for displaying PNGs in case-page
- [ ] Add VAF to cancer case general report, and remove some of its unused fields
- [ ] Variants filter compatible with genome browser location strings
- [ ] Support for custom public igv tracks stored on the cloud
- [ ] Add tests to increase testing coverage
- [ ] Update case variants count after deleting variants
- [ ] Update IGV.js to latest (v2.7.4)
- [ ] Bypass igv.js CORS check using `https://github.com/Rob--W/cors-anywhere`
- [ ] Documentation on default and custom IGV.js tracks (admin docs)
- [ ] Lock phenomodels so they're editable by admins only
- [ ] Small case group assessment sharing
- [ ] Tutorial and files for deploying app on containers (Kubernetes pods)
- [ ] Canonical transcript and protein change of canonical transcript in exported variants excel sheet
- [ ] Support for Font Awesome version 6
- [ ] Submit to Beacon from case page sidebar
- [ ] Hide dismissed variants in variants pages and variants export function
- [ ] Systemd service files and instruction to deploy Scout using podman
### Fixed
- [ ] Bugfix: unused `chromgraph_prefix |tojson` removed
- [ ] Freeze coloredlogs temporarily
- [ ] Marrvel link
- [ ] Don't show TP53 link for silent changes
- [ ] OMIM gene field accepts any custom number as OMIM gene
- [ ] Fix Pytest single quote vs double quote string
- [ ] Bug in gene variants search when providing similar case display name
- [ ] Delete unused file `userpanel.py`
- [ ] Primary transcripts in variant overview and general report
- [ ] Google OAuth2 login setup in README file
- [ ] Redirect to 'missing file'-icon if configured file is missing
- [ ] Javascript error in case page
- [ ] Fix compound matching during variant loading for hg38
- [ ] Cancer variants view containing variants dismissed with cancer-specific reasons
- [ ] Zoom to SV variant length was missing IGV contig select
- [ ] Tooltips on case page when case has no default gene panels
### Changed
- [ ] Save case variants count in case document and not in sessions
- [ ] Style of gene panels multiselect on case page
- [ ] Collapse/expand main HPO checkboxes in phenomodel preview
- [ ] Replaced GQ (Genotype quality) with VAF (Variant allele frequency) in cancer variants GT table
- [ ] Allow loading of cancer cases with no tumor_purity field

**Comment by northwestwitch** [comment]
All fixed, I'd merge and deploy..

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,7 +39,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Delete unused file `userpanel.py`
 - Primary transcripts in variant overview and general report
 - Google OAuth2 login setup in README file
-- Redirect to 'missing file'-icon if configured file is missing
+- Redirect to 'missing file'-icon if configured chromogram file is missing
```
Comment (dnil) [inline CHANGELOG.md:42]: Chromograph
```suggestion
- Redirect to 'missing file'-icon if configured Chromograph file is missing
```

**Review by dnil — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md]: Yes, better!

**Comment by northwestwitch** [comment]
Deployed on stage and prod!

image

**Pull request merged**